src/metrics/coefficientofdetermination.jl

```diff
@@ -1,6 +1,6 @@
 import StatsBase
 
-function R2coefficientofdetermination(
+function R2_coefficient_of_determination(
```

Comment:
I feel that `r2_score` or `r_squared` are sufficient. Typically people may now that it refers to coefficient of determination, and if they don't that's what doc_strings are for

```
"""
 r2_score()
Computes coefficient of determination
"""
```

Reply:
r2_score with a docstring looks good to me.

Reply:
Done.